prow/cmd/deck/static/script.js

```diff
@@ -94,7 +94,16 @@ function groupKey(build) {
     return build.repo + " " + build.number + " " + build.refs;
 }
 
+function configure() {
+    if(typeof config.deck === 'undefined') || (typeof config.deck.branding === 'undefined'){
+        return;
+    }
+    document.getElementById("img").src = config.deck.branding.logo;
+    document.getElementById("favicon").href = config.deck.branding.favicon;
```

Comment:
Only set these if they are non-empty so that one can be specified without the other.